packages/address/src/tests/utilities.test.ts

```diff
@@ -1,10 +1,18 @@
+import {Address} from '@shopify/address-consts';
 import {fixtures} from '@shopify/address-mocks';
 
-import {renderLineTemplate} from '../utilities';
+import {
+  buildOrderedFields,
+  formatAddress,
+  renderLineTemplate,
+} from '../utilities';
 
 const Canada = fixtures.country.EN.data.country;
+const Japan = fixtures.countries.JA.data.countries.find(
+  ({code}) => code === 'JP',
+)!;
 
-const address = {
+const address: Address = {
```

Comment:
Explicitly typing literals ensure that no extra properties are defined, like typos.

Reply:
We could also do `Parameters[typeof formatAddress][0]` to avoid the extra import. This also describes how this it is intending to be used.

Reply:
I'm not sure I see the benefit 🤔 

To me, it feels more confusing and tightly coupled than the simple independent fixture object it is supposed to be. There's also a similar fixture object typed using `Address` in the `AddressFormatter.test.ts` file.

If there's any concern I did not anticipate, should we change both test files?